code/WorkInProgress/AbilityItem.dm

```diff
@@ -489,47 +510,49 @@
 			else
 				usr.client.view = "15x15"
 				icon_state = "airoff"
+		..()
 
 ////////////////////////////////////////////////////////////
 
 /* this version moves the center of your view */
 
 /obj/ability_button/scope_toggle
```

Comment:
Not sure if this stuff will compile due to indentation.

Reply:
Indentation looks fine to me?

Reply:
indentation is fine and it compiles, i want to call back to parent if either conditional runs, not just if only one or the other runs